app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -219,6 +220,29 @@ func (l *LoadBalancerSyncer) PrintStatus() (string, error) {
 	return fmt.Sprintf("Load balancer %s has IPAddress %s and is spread across %d clusters (%s)", l.lbName, sd.IPAddress, len(sd.Clusters), strings.Join(sd.Clusters, ",")), nil
 }
 
+func formatForwardingRules(rules []status.LoadBalancerStatus) (string, error) {
```

Comment:
how about calling this formatLoadBalancersList? This method does not know that load balancer statuses came from forwarding rules and hence should have nothing to do with forwarding rules.